bp_common/test/src/bootrom/Makefile

```diff
@@ -1,8 +1,26 @@
 
-RISCV_GCC = $(CROSS_COMPILE)gcc -fPIC -march=rv64im -mabi=lp64 -mcmodel=medany -static -nostdlib -nostartfiles
+RISCV_GCC = $(CROSS_COMPILE)gcc -march=rv64im -mabi=lp64 -mcmodel=medany -static -nostdlib -nostartfiles
 
-build:
-	$(RISCV_GCC) bootrom.S -o bootrom.riscv -Tlink.ld -static -Wl,--no-gc-sections
+UCODE2BOOT = $(BP_COMMON_DIR)/software/py/ucode2boot.py
+
+ifdef CCE_MEM_PATH
+BOOT_INPUTS = --ucode=$(CCE_MEM_PATH)/$(CCE_MEM)
```

Comment:
What's this used for?

Reply:
This is variable is useful when the default bootrom is being generated as part of `make prep_lite` or `make prep`. During the actual simulation, the user might want to switch the protocol, so the target changes the input to the script when doing this as part of the simulation flow. Now, I am not sure if we still completely support the other protocols or we have moved away from it, but I was just "thinking ahead"

Reply:
Ah, gotcha.  We don't actually want to generate the bootrom at make prep time, since the default should be committed to the repo.  So probably don't need this